doc/integration/rest-assured/rest.md

```diff
@@ -28,6 +28,8 @@ rest { # there you describe configuration for default rest context
 
   # base url which will be prefixed for every request
   base-url = "http://myhost:port/path"
+  socket-timeout = 30000 #Socket timeout in milliseconds (optional)
```

Comment:
You can use
```kotlin
config.getDuration("socket-timeout", TimeUnit.MILLISECONDS)
```
to obtain value in millis from human-readable config value (i.e. `10 seconds`)

Reply:
Unfortunatelly, getDuration returns a Long value, but Int is needed.